boards/tutorials/nrf52840dk-thread-tutorial/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+# Licensed under the Apache License, Version 2.0 or the MIT License.
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+# Copyright Tock Contributors 2022.
+
+[package]
+name = "nrf52840dk-thread-tutorial"
+version.workspace = true
+authors.workspace = true
+build = "../../build.rs"
+edition.workspace = true
+
+[features]
+default = ["screen_ssd1306"]
+screen_ssd1306 = []
+screen_sh1106 = []
```

Comment:
Cargo features just do not work for us do they.

Reply:
:confused: I know, do you feel like we should remove this? Really this is only such that we can still easily support users with the wrong model / develop using the other screen that we have.